File: app/src/main/java/com/app/rupiksha/models/WalletToWalletData.java
package com.app.rupiksha.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WalletToWalletData
{
    @SerializedName("date")
    @Expose
    private String date;
    @SerializedName("senderName")
    @Expose
    private String senderName;
    @SerializedName("senderMobile")
    @Expose
    private String senderMobile;
    @SerializedName("receiverName")
    @Expose
    private String receiverName;
    @SerializedName("receiverMobile")
    @Expose
    private String receiverMobile;
    @SerializedName("txnid")
    @Expose
    private String txnid;
    @SerializedName("amount")
    @Expose
    private String amount;
    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("support")
    @Expose
    private String support;

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getSenderName() {
        return senderName;
    }

    public void setSenderName(String senderName) {
        this.senderName = senderName;
    }

    public String getSenderMobile() {
        return senderMobile;
    }

    public void setSenderMobile(String senderMobile) {
        this.senderMobile = senderMobile;
    }

    public String getReceiverName() {
        return receiverName;
    }

    public void setReceiverName(String receiverName) {
        this.receiverName = receiverName;
    }

    public String getReceiverMobile() {
        return receiverMobile;
    }

    public void setReceiverMobile(String receiverMobile) {
        this.receiverMobile = receiverMobile;
    }

    public String getTxnid() {
        return txnid;
    }

    public void setTxnid(String txnid) {
        this.txnid = txnid;
    }

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getSupport() {
        return support;
    }

    public void setSupport(String support) {
        this.support = support;
    }
}

File: app/src/main/java/com/app/rupiksha/models/WalletToWalletData.kt
package com.app.rupiksha.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class WalletToWalletData(
    @SerializedName("date")
    @Expose
    var date: String? = null,

    @SerializedName("senderName")
    @Expose
    var senderName: String? = null,

    @SerializedName("senderMobile")
    @Expose
    var senderMobile: String? = null,

    @SerializedName("receiverName")
    @Expose
    var receiverName: String? = null,

    @SerializedName("receiverMobile")
    @Expose
    var receiverMobile: String? = null,

    @SerializedName("txnid")
    @Expose
    var txnid: String? = null,

    @SerializedName("amount")
    @Expose
    var amount: String? = null,

    @SerializedName("status")
    @Expose
    var status: String? = null,

    @SerializedName("support")
    @Expose
    var support: String? = null
)
